### BIG differences, this is diff of wrong files.
### CLUS "Macro LAMBDA"
### and CLHS topic selection http://clhs.lisp.se/Body/a_lambda.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:macros:lambda.txt	2017-04-27 21:57:52.000000000 +0100
+++ saved-pages-clus/cl:macros:lambda.txt	2017-04-27 21:57:53.000000000 +0100
@@ -1,2 +1,1 @@
-LAMBDA
-Please select which reference to LAMBDA you intended
+Macro LAMBDA

-MacroSymbol
\ No newline at end of file
+Syntax
+ lambda lambda-list [[declaration* | documentation]] form*
+→ function
+Arguments and Values
+ lambda-list - an ordinary lambda list.
+ declaration - a declare expression; not evaluated.
+ documentation - a string; not evaluated.
+ form - a form.
+ function - a function.
+Description
+Provides a shorthand notation for a function special form involving a lambda expression such that
+ (lambda lambda-list [[declaration* | documentation]] form*)
+  ≡ (function (lambda lambda-list [[declaration* | documentation]] form*))
+  ≡ #'(lambda lambda-list [[declaration* | documentation]] form*)
+
+Examples
+(funcall (lambda (x) (+ x 3)) 4)
+→
+7
+Side Effects
+None.
+Affected By
+None.
+Exceptional Situations
+None.
+See Also
+Symbol LAMBDA
+Notes
+This macro could be implemented by
+(defmacro lambda (&whole form &rest bvl-decls-and-body)
+  (declare (ignore bvl-decls-and-body))
+  `#',form)
\ No newline at end of file
